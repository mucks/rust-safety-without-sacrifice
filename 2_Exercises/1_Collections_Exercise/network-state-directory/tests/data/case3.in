# Prefix filtering
MEMBER ForestCity 1 Alice Citizen
MEMBER ForestCity 2 Akira Validator
MEMBER ForestCity 3 Ben Citizen
MEMBER ForestCity 4 Ana Founder
PREFIX ForestCity A
STATS
