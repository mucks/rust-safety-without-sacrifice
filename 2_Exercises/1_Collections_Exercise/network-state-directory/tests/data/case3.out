PREFIX ForestCity A:
 - Alice (1)
 - Akira (2)
 - Ana (4)
STATS citizens=2 validators=1 founders=1
